frontend/.gitignore

```diff
@@ -13,6 +14,7 @@ dist-ssr
 *.local
 
 # Editor directories and files
+.env
```

Comment:
This file is already included above in the file, making this line redundant. Remove this.